src/d3a/d3a_core/cli.py

```diff
@@ -80,9 +80,10 @@ def main(log_level):
                   ', '.join(_setup_modules)))
 @click.option('-g', '--settings-file', default=None,
               help="Settings file path")
-@click.option('--slowdown', type=int, default=0,
+@click.option('--slowdown', type=float, default=0,
               help="Slowdown factor [0 - 10,000]. "
-                   "Where 0 means: no slowdown, ticks are simulated as fast as possible; "
+                   "Where 0 means: no slowdown, ticks are simulated as fast as possible;"
+                   "You can give float value also which makes simulation slower"
```

Comment:
This can be skipped as the user would already know via `d3a run --help` that `slowdown` can now be configured as float value

Reply:
Faizen you mean to skip which line ? in the above change?....user can enter int value and also float value.

Reply:
Changed the comments to more accurate... "You can give float value also which makes simulation slower"